readme.md

```diff
@@ -81,9 +113,12 @@ var Panda = function(name, age) {
 ```
 
 Your Answer:
-```text
-
 ```
+var eat_bamboo = function () {
+  console.log('this.name' + ('num_bamboo_eaten' + ii'));
```

Comment:
No need to console log here. Check the solution branch to see a correct version. The important things:
* the function needs to be defined on the prototype so that each panda can use the function.
* the function should just update the `this.num_bamboo_eaten` property.